.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ on:
 
 jobs:
   build:
-    continue-on-error: ${{ contains(fromJSON('["3.7", "3.12-dev", "pypy3.7", "pypy3.10"]'), inputs.python-version) }}
+    continue-on-error: ${{ endsWith(inputs.python-version, '-dev') || contains(fromJSON('["3.7", "pypy3.7", "pypy3.10"]'), inputs.python-version) }}
```

Comment:
Do we really want to generalize this? I would hope that not all version bumps carry that much baggage as the 3.12 one.

Reply:
I think it would be desirable to switch on CI with future alphas ASAP, however I'd make a bet that ffi-check (at minimum) will break with every new alpha version, and maybe other test failures. I'd personally prefer this because it should provide a nudge to make small fixups through the year as each alpha / beta release drops rather than what's happened this year where PyO3 has fallen relatively far behind on 3.12 beta compatibility.

If future version bumps are trivial then generalizing like this is not a big deal; on the other hand if a future version bump is nasty, then generalizing like this will minimize blocking failures for other contributors :)

Reply:
Fair enough.